web/src/helpers/cadPackages/openScadController.js

```diff
@@ -2,17 +2,22 @@ let openScadBaseURL =
   process.env.OPENSCAD_BASE_URL ||
   'https://x2wvhihk56.execute-api.us-east-1.amazonaws.com/dev'
 
+let lastCameraSettings
+
 export const render = async ({ code, settings }) => {
   const body = JSON.stringify({
     settings: {
       size: {
         x: 500,
         y: 500,
       },
-      camera: settings.camera,
+      camera: settings.camera || lastCameraSettings,
     },
     file: code,
   })
+  if (settings.camera) {
+    lastCameraSettings = settings.camera
```

Comment:
Persisting the last cameraSettings, so that this function can be called without always having to provide camera settings, Currently it relies on the viewer making the first call so that it's not completely empty. I should probably put some default values in here where the variable is defined . . . hmmm